src/tools/wasm-interp.cc

```diff
@@ -39,6 +40,19 @@
 using namespace wabt;
 using namespace wabt::interp;
 
+struct FunctionCallee {
```

Comment:
The naming if this struct seems a little odd.  How about just just `FunctionCall` since this seems like more than just the callee and also includes a set of argument bindings

Reply:
Hmm, make sense. Wanted to rename the old struct from the reference PR but also agree that Callee is off. Updated [0fcea09](https://github.com/WebAssembly/wabt/pull/2176/commits/0fcea09a9c5edc4daf130bbf81b08b837ebe734c)